workflows/machine_data/live/write/create_and_connect_machine_to_site/create_and_connect_machine_to_site.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+## Shell file to stream information about the live machine kinematic state.
+
+## Settings for the environment.
+env="qa"
+dc="us"
+site_id=""
+
+## Authorization. OAuth credentials are used if the JWT string is empty.
+## run `SitelinkFrontend.core.store.getState().app.owner.jwt[0]` in your browser developer console to obtain a JWT.
+jwt=""
+# - or -
+oauth_id=""
+oauth_secret=""
+oauth_scope=""
+
+machine_resource_configuration_file="resource_configurations/haul_truck.resource_configuration.json"
```

Comment:
where is haul_truck.resource_configuration.json?, I would like to confirm it is using wgs for replicate transforms as it is indicated in the mfk replicate function make_haul_mfk_replicate_payload.

Reply:
Sorry, missed commit of the resource configuration.